src/pages/documentation/building-mobile-extensions/interacting-with-rules.md

```diff
@@ -1,95 +0,0 @@
-# Interacting with rules
-
-Rules are defined in the Data Collection UI interface and can include data elements, events, conditions, and actions provided by extensions.
-
-## Publishing rules support for tags
-
-Your extension should publish the rules support that it provides to tag libraries. You can provide this support through the `extension.json` file in your extensions base directory.
-
-### Publishing supported data elements
```

Comment:
do we have some rule examples for extensions in the github docs?

Reply:
Not currently but most of this is covered in the https://experienceleague.adobe.com/docs/experience-platform/tags/extension-dev/overview.html?lang=en

Reply:
there are few more details about how to create data elements, events etc for extensions, maybe we can preserve some of this in github / here